src/components/VideoRating.tsx

```diff
@@ -0,0 +1,37 @@
+import "./VideoRating.scss";
+import Rating from "./../assets/reiting.png";
+import { StarRating } from "./svg/StarRating";
+import { useState } from "react";
+
+const VideoRating = () => {
+  const [rating, setRating] = useState(0);
+  const [hover, setHover] = useState(0);
+  
+  return (
+    <div className="star-rating">
+      <br/>
+      {[...Array(5)].map((star, index) => {
+        index += 1;
```

Comment:
You need create here a new const. It is an anti pattern to mutate an argument inside a function:

const const ratingUnit = index + 1;

And then use it ratingUnit in your button

Reply:
ok